xml/osc_building.xml

```diff
@@ -140,6 +141,62 @@ The buildroot was: /var/tmp/build-root/openSUSE_Tumbleweed-x86_64</screen>
   </procedure>
  </sect1>
 
+ <sect1 xml:id="sec.obs.building.advanced">
+  <title>Advanced Build Environment Handling</title>
+  <para>
+	  The default build environment for local builds is usually chroot. While this is simplest environment
+	  and is therefore easy and fast to handle, it has also a number of shortcummings. For one it is
+	  not safe against attacks, so you must not build sources or using build dependendencies from a resource
+	  which you do not trust. Furthermore the environment is not fully isolated and runs not the kernel the
+	  target distribution runs. This means esp image builds and kernel/hardware specific builds may fail or
+	  won't produce the same result. The server side is usually set to inside of KVM therefore to avoid this.
+  </para>
+  <itemizedlist>
+   <listitem>
+     <para>You can build also local in KVM (if your hardware supports it) by running
+      </para>
+      <screen>&prompt.user;<command>osc</command> build --vm-type=kvm --vm-memory=<replaceable>MB</replaceable></screen>
+   </listitem>
+   <listitem>
+      <para>Another important virtualization mode is qemu. This can be used to build for a foreign hardware
+	      architecture even when the distribution is not prepared for the qemu user land emulator. However, this
+	      qemu system emulator approach will be much slower. 
+      </para>
+      <screen>&prompt.user;<command>osc</command> build --vm-type=qemu --vm-memory=<replaceable>MB</replaceable> <replaceable>REPOSITORY</replaceable> <replaceable>ARCHITECTURE</replaceable></screen>
+   </listitem>
+   <listitem>
+     <para>You may want to jump inside of a the build environment for debugging purposes. This can be done via the following
```

Comment:
```suggestion
     <para>You may want to jump inside of a the build environment for debugging purposes. This can be done via the following command:
```